openproblems/tasks/batch_integration_graph/datasets/pancreas.py

```diff
@@ -0,0 +1,31 @@
+from ....data.pancreas import load_pancreas
+from ....tools.decorators import dataset
+from scIB.preprocessing import normalize, hvg_batch
+
+import numpy as np
+import scanpy as sc
+
+
+@dataset("Pancreas (by batch)")
+def pancreas_batch(test=False):
+    adata = load_pancreas(test=test)
+    adata.obs["labels"] = adata.obs["celltype"]
+
+    adata.obs["batch"] = adata.obs["tech"]
+
+    normalize(adata)
+    
+    hvg_list = hvg_batch(adata, batch_key='batch')
+    adata.var['highly_variable'] = np.in1d(adata.var_names, hvg_list)
```

Comment:
We should consider making HVG selection part of the method itself and use this like normalization. The eventual data loader will not include normalization.